lib/timer.js

```diff
@@ -19,8 +19,8 @@ class Timer {
    */
   handler(key, schedule, listener) {
     const { interval, cron, cronOptions, immediate } = schedule;
-    if (!interval && !cron) {
-      throw new Error('[egg-schedule] schedule.interval or schedule.cron must be present');
+    if (!interval && !cron && !immediate) {
```

Comment:
I had another idea, we should change this check to `assert`.

```js
assert(interval || cron || immediate, '[egg-schedule] schedule.interval or schedule.cron or schedule.immediate must be present')
```

Reply:
ok, it's well.